Wrapping/Generators/Python/itk/support/helpers.py

```diff
@@ -40,6 +40,18 @@
     pass
 
 
+def snake_to_camel_case(keyword: str):
```

Comment:
By renaming `_snake_to_camel` to `snake_to_camel_case`, it has become part of the public `itk` Python API, right? Just checking 🤓

Reply:
Yes :-)